third_party/nvfuser/csrc/lower_expr_sort.cpp

```diff
@@ -1189,13 +1189,30 @@ bool ExprSegmentationSorter::supportedMerge(ExprGroup* sg1, ExprGroup* sg2) {
     return false;
```

Comment:
Can you pull these changes out?

Reply:
I reverted an incorrect merge.  Thanks for catching this!